package.json

```diff
@@ -28,39 +28,40 @@
   },
   "dependencies": {
     "ember-auto-import": "^1.2.19",
-    "ember-cli-babel": "^7.1.2",
+    "ember-cli-babel": "^7.11.1",
+    "ember-cli-htmlbars": "^4.0.0",
     "simple-pinyin": "^3.0.2",
     "speakingurl": "^14.0.1",
     "unorm": "^1.5.0"
   },
   "devDependencies": {
-    "@ember/optional-features": "^0.6.3",
-    "broccoli-asset-rev": "^2.7.0",
+    "@ember/optional-features": "^1.0.0",
+    "babel-eslint": "^10.0.3",
+    "broccoli-asset-rev": "^3.0.0",
     "ember-circleci": "^1.2.0",
-    "ember-cli": "~3.8.3",
+    "ember-cli": "~3.13.1",
     "ember-cli-dependency-checker": "^3.1.0",
-    "ember-cli-eslint": "^4.2.3",
-    "ember-cli-htmlbars": "^3.0.0",
-    "ember-cli-htmlbars-inline-precompile": "^1.0.3",
-    "ember-cli-inject-live-reload": "^1.8.2",
+    "ember-cli-eslint": "^5.1.0",
+    "ember-cli-htmlbars-inline-precompile": "^3.0.0",
+    "ember-cli-inject-live-reload": "^2.0.1",
     "ember-cli-sri": "^2.1.1",
-    "ember-cli-template-lint": "^1.0.0-beta.1",
-    "ember-cli-uglify": "^2.1.0",
+    "ember-cli-template-lint": "^1.0.0-beta.3",
+    "ember-cli-uglify": "^3.0.0",
     "ember-disable-prototype-extensions": "^1.1.3",
     "ember-export-application-global": "^2.0.0",
-    "ember-load-initializers": "^1.1.0",
+    "ember-load-initializers": "^2.1.0",
     "ember-maybe-import-regenerator": "^0.1.6",
-    "ember-qunit": "^3.4.1",
-    "ember-resolver": "^5.0.1",
-    "ember-source": "~3.8.0",
-    "ember-source-channel-url": "^1.1.0",
-    "ember-try": "^1.0.0",
+    "ember-qunit": "^4.5.1",
+    "ember-resolver": "^5.3.0",
+    "ember-source": "~3.13.0",
+    "ember-source-channel-url": "^2.0.1",
+    "ember-try": "^1.2.1",
     "eslint-config-peopledoc": "^1.5.0",
```

Comment:
Did you plan to update in a separate PR ?

Reply:
you are right, might as well be dont now